tests/test.bal

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/config;
+import ballerina/log;
+import ballerina/test;
+import ballerina/time;
+
+CalendarConfiguration config = {
+    oauth2Config: {
+        accessToken: config:getAsString("ACCESS_TOKEN"),
+        refreshConfig: {
+            clientId: config:getAsString("CLIENT_ID"),
+            clientSecret: config:getAsString("CLIENT_SECRET"),
+            refreshUrl: config:getAsString("REFRESH_URL"),
+            refreshToken: config:getAsString("REFRESH_TOKEN")
+        }
+    }
+};
+
+CalendarClient calendarClient = new(config); 
+
+string testEventId = "";
+string testChannelId = "";
+string testResourceId = "";
+string testQuickAddEventId = "";
+
+CreateEventOptional optional = {
+    conferenceDataVersion: 1,
+    sendUpdates: "all",
+    supportsAttachments: false
+};
+
+@test:Config{}
+function testCreateEvent() {
+    InputEvent|error event = setEvent("Event Created");
+    if (event is InputEvent) {
+        log:print("calendarClient -> createEvent()");
```

Comment:
use log:info

Reply:
printInfo is removed in slp8. So we have to use print.